src/CSI.IBTA.DataLayer/Interfaces/IGenericRepository.cs

```diff
@@ -1,14 +1,14 @@
 ﻿using System.Linq.Expressions;
 
-namespace CSI.IBTA.Shared.Interfaces
+namespace CSI.IBTA.DataLayer.Interfaces
 {
     public interface IGenericRepository<T> where T : class
     {
         Task<IEnumerable<T>> All();
         Task<T?> GetById(int id);
```

Comment:
Worse to mention that id is not necessary to be int, in general it should be generic. 
For now, lets go with what we have and try to use only int for PKs in DB.